README.md

```diff
@@ -22,6 +27,9 @@ $ cat PROXIES.txt | python main.py --realtime
 ```
 
 #### `PROXIES.txt` should be in following format (one proxy per line):
+#### IMPORTANT NOTE
```

Comment:
Similarly to above, let's use bolded NOTE section, and not make the text very big.